packages/core/database/lib/validations/relations/bidirectional.js

```diff
@@ -32,7 +32,7 @@ const getLinksWithoutMappedBy = (db) => {
 
 const isLinkTableEmpty = async (db, linkTableName) => {
   // If the table doesn't exist, it's empty
-  const exists = await db.getConnection().schema.hasTable(linkTableName);
+  const exists = await db.getSchemaConnection().hasTable(linkTableName);
```

Comment:
Forgot we had  getSchemaConnection. Good catch!

Reply:
Just a random question: why is this not catched by any test? :)

Reply:
That is a very good question and a thing we could discuss better.

This is caused by a custom configuration on postgres we didnt account for. For the default config is working ok.

We have faced multiple issues caused by db configs

GTID issues
Mariadb clusters
Postgress custom schemas

We could extend a list and periodically test for it, or see if there is a possibility to add it in the CI.

It is quite complicated to account for things like this in our current tests today.

Reply:
Interesting. I imagine it would be quite hard to run all tests against every combination of configuration. Do you see a way to create tests, e.g. to mimic a specific configuration? E.g. to mock something in a way to simulate such a configuration?